CC/Sounder/comms-lib.cc

```diff
@@ -584,6 +638,7 @@ std::vector<std::vector<double>> CommsLib::getSequence(int N, int type)
             -0.0040763264805083466, 0.025888347648318433, 0.10617091261510256,
             0.05518049537437035, 0.08770675983572167, -0.027885918828227545,
             -0.08279790948776067, 0.11115794305116433, 0.12032513267372755 };
+	*/
```

Comment:
Can we remove the commented part?

Reply:
Sure. I had this part commented out as I wanted to make sure the new implementation is working properly. If you think it's okay I can remove the old hard-coded STS and LTS.

Also note that I am not sure where did the hard-coded LTS come from but the I/Q samples seem to be different that that I generated following the 11a standards (maybe it was a different set of OFDM symbols?)

Reply:
I think the hardcoded values were just scaled versus the one in the standard. Some scaling would be nice to avoid clipping. But `get_sequence` should generate the original version.

Is the code you have added tested on hardware?

Reply:
Yes, the generated STS and LTS have been tested on USRP SDRs, see example over-the-air transmissions. By default using CommsLib::IFFT the maximum I/Q amplitude should be less than 0.5.

<img width="774" alt="usrp_sts_gold" src="https://user-images.githubusercontent.com/12038706/98322793-4a8a0400-1fb6-11eb-83c2-f6e1d1dfee93.png">
<img width="772" alt="usrp_lts" src="https://user-images.githubusercontent.com/12038706/98322798-4c53c780-1fb6-11eb-84d6-e2d7e0f4847b.png">